framer/LayerStates.coffee

```diff
@@ -179,7 +178,7 @@ class exports.LayerStates extends BaseClass
 		# TODO: Maybe we want to support advanced data structures like objects in the future too.
 		for k, v of properties
 
-			if _.isString(v) && Color.isColorString(v)
+			if _.isString(v) && _.endsWith(k.toLowerCase(),'color') && Color.isColorString(v)
```

Comment:
- We use `"` in the Framer source. 
- We favor `and` over && in cs.

Ideally I would add some comments here why we check the key name.